samples/pipeline-tests.sh

```diff
@@ -1,110 +1,196 @@
+# usage: ./pipeline-tests.sh [sleepTime] [kubectl|seldon] [namespace]
+
 if [ -z "$1" ]
 then
       sleepTime=5
 else
       sleepTime=$1
 fi
 
+if [ -z "$2" ]
+then
+      cmd="kubectl"
+else
+      cmd=$2
+fi
+
+if [ -z "$3" ]
+then
+      namespace="seldon-mesh"
+else
+      namespace=$3
+fi
+
+function load() {
```

Comment:
Is the goal here to not need to set `seldon` environment variables pointing to the k8s svc? If yes perhaps we should also have the equivalent of `seldon status` commands using kubectl.

Reply:
good point, I will add it.

Reply:
done in [70285e0](https://github.com/SeldonIO/seldon-core/pull/5307/commits/70285e0ae1d4a3a992ea4090659fbf78e79d8871)